topdown/net_test.go

```diff
@@ -155,6 +155,63 @@ func TestNetLookupIPAddr(t *testing.T) {
 			}
 		})
 	}
+
+	addr := "v4.org"
+	exp := ast.NewSet(ast.StringTerm("1.2.3.4"))
+	for name, allowNet := range map[string][]string{
+		"allow_net nil":                     nil,
+		"allow_net match":                   {addr},
+		"allow_net match + additional host": {addr, "example.com"},
+	} {
+		t.Run(name, func(t *testing.T) {
+			ctx, cancel := context.WithCancel(context.Background())
```

Comment:
[nit] Do we somehow need the cancel here? We night just do `ctx := context.Background()`; but I might be missing something.

Reply:
No, I thing you're right. This is a copy/paste I didn't clean up properly.

Reply:
Hm, all other test cases use cancel. I'll keep this as-is, just to be in line with the rest of them. However, this might be unnecessary in all cases.